content/02.body.md

```diff
@@ -2,13 +2,65 @@
+Reliance on degree may be unfavorable depending on the prediction/analysis task being conducted and the magnitude of possible degree bias in the data.
+
+Another reason why a reliance on degree can be unfavorable is that degree-based predictions may be nonspecific.
+Nonspecific predictions are based on information that, while potentially associated with true connections, is not reflective of the underlying causal relationships.
+For example, predicting that the highest degree node in a network shares edges with the remaining nodes to which it is not connected will often lead to many correct predictions, despite the fact that the predictions were generic to all nodes in the network.
+Gillis et al. [@doi:10.1371/journal.pone.0017258] explored the concept of prediction specificity with respect to gene function prediction, in which predictions based on gene multifunctionality could be, "potentially misleading with respect to causality."
+The authors found that, "[N]ode degree is unambiguously linked to multifunctionality," and proposed that, "[T]here are no computational gene function assignments that can be shown to preclude the possibility they reflect only underlying multifunctionality."
+We believe that edge prediction nonspecificity due to node degree imbalance is a general phenomenon across other types of data.
+
+Real networks have a variety of degree distributions, and they commonly exhibit degree imbalance [@doi:10.1371/journal.pone.0017645; @doi:10.1007/978-1-61779-361-5_13; @doi:10.1038/s41467-019-08746-5].
+Figure {@fig:degree}D shows examples of different degree distributions in six biomedical networks.
+Degree imbalance leads high-degree nodes to dominate in the predictions made by degree-associated methods [@doi:10.1093/bioinformatics/btv215], which have been shown to be effective predictors of connections in some biological networks [@doi:10.1186/1752-0509-2-11].
+Consequently, degree-based predictions are more likely nonspecific, meaning the same set of predictions would appear to perform well for different tasks.
+However, depending on the prediction task and the desired types of predictions, edge predictions between very high degree nodes may be uninsightful or nonspecific.
+Model evaluation can also be complicated, as nonspecific or trivial predictions need not be incorrect, even if they are not the desired outputs of the predictive model.
+Rather, degree-based features must be included in the interpretation of predictions in order to disentangle desired from non-desired effects and to more effectively evaluate and compare predictive models.
+
+![**A.** Degree distributions of networks with and without degree bias can be very different.
+  Protein-protein interaction (PPI) literature-derived network from STRING (Szklarczyk et al.) [@doi:10.1093/nar/gky1131], high-throughput-derived networks from Rual et al. [@doi:10.1038/nature04209] and Rolland et al. [@doi:10.1016/j.cell.2014.10.050].
```

Comment:
No need for `Szklarczyk et al.` when you cite the publication and name the RESOURCE.